### DISCUSSION TITLE
Bibliography in Annex

### DISCUSSION THREAD

**Discussion by AnaaCorreia** [joaomlourenco/novathesis#343] [Q&A] [answered]
Hello, 
(Version 2021-09-11 [6.5.1])

I have bibliography per chapter. 

In annex 1, I need to cite some references and consequently include its bibliography in the end of the annex. Despite its compilation (I can see the numbers during the text), it doesn't appear in the end. Should I do something different?

I'm using the code:

\begin{refsection}

(....)

\printbibliography[heading=subbibliography, segment=\therefsegment, title={\bibname\ for Annex~\thechapter}]

\end{refsection}

Thanks in advance,

**Comment by AnaaCorreia** [comment]
Any suggestion?

Thanks in advance.

**Comment by joaomlourenco** [comment, marked as answer]
That's a bug that has been fixed meanwhile.
Please update to the latest version of the template.
